migrations/20200604110145_create_commit_table.ts

```diff
@@ -0,0 +1,20 @@
+import * as Knex from "knex";
+
+const commitTableName = "commit";
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.createTable(
+    commitTableName,
+    (builder: Knex.TableBuilder) => {
+      builder.increments("id").primary();
```

Comment:
For commits, we can use commit hash as [natural key](https://en.wikipedia.org/wiki/Natural_key), so we don't need to use an auto-increment integer as surrogate key.